### PR TITLE
[MIRROR] adds #cargobus and gauntlets (also modernizes ntchat)

### PR DESCRIPTION
**Pull request by SkyratBot**
Original PR: https://github.com/tgstation/tgstation/pull/58609
--------------------

## About The Pull Request

Removes Request consoles from the outside of cargo, you may still have someone from inside cargo order things for you

Request consoles are now in each department, with a modular console next to it logged in as a department account in an irc channel called #cargobus

- [x] Finish Mapping this
~Possibly consider making automatic forms of delivery harder to encourage manual deliveries~ - not for this pr
- [x] Cargo techies's tablets start with cargobus as well
- [x] Make #cargobus a bit stronger (remove the ability to delete or rename it)

Cargo now starts with a few pairs of H.A.U.L. Gauntlets that give you the ability to have VERY strong pulls! Nobody can break your pulls by grabbing it out from under you

- [x] Consider Making lockers and crates fully unable to be opened while strong pulled
- [x] Add signals for when you stop pulling, to unregister signals

## Why It's Good For The Game

Cargo in it's current state has dropped a lot of the "delivery" aspect of cargo, instead opting for people coming to cargo to get what they want when it should be the other way around. Why? Because it's a pain to drag crates where they need to be if everyone's going to try and open it or steal it when you do. HAUL gauntlets were designed to help with that. The #cargobus and moving of request consoles creates distance between the place you request to get an order and where cargo recieves that order to further encourage cargo having to deliver shit.

tl;dr cargo does a lot of scavenging, but has little to no interaction with 

## Changelog
:cl:
add: Cargo requests can now be discussed over #cargobus!
/:cl:

### CODE REVIEW

**Review by KathrinBailey — APPROVED** [review]
paw
